src/Microsoft.Azure.SignalR/Utilities/AuthorizeHelper.cs

```diff
@@ -66,5 +67,21 @@ public static async Task<bool> AuthorizeAsync(HttpContext context, IList<IAuthor
             }
             return false;
         }
+
+        public static List<IAuthorizeData> BuildAuthorizePolicy(Type hub)
```

Comment:
You can also remove the ifdef here.

Reply:
Users may still call UseAzureSignalR() in Core 3.0 which will build ServiceRouteBuilder and then come here. So better to add a check here to skip it completely under 3.0.